s3tests/functional/test_s3.py

```diff
@@ -8266,3 +8266,114 @@ def test_sse_kms_barb_transfer_13b():
     if 'kms_keyid' not in config['main']:
         raise SkipTest
     _test_sse_kms_customer_write(13, key_id = config['main']['kms_keyid'])
+
+@attr(resource='bucket')
+@attr(method='get')
+@attr(operation='Test Bucket Policy')
+@attr(assertion='succeeds')
+@attr('bucket-policy')
+def test_bucket_policy():
+    bucket = get_new_bucket(name='my-bucket')
```

Comment:
Well, the functional tests give a provision to create a bucket of a specific name, and I need to use the name further down in the test code, hence I have used it.